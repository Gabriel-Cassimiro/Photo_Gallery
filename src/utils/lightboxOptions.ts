export const Options = {
	settings: {},
	caption: {},
	buttons: {},
	thumbnails: {},
	progressBar: {}
}
