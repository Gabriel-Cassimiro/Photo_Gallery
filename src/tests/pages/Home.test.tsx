/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react"
import Home from "../../pages/index"
import { TestProvider } from "../../tests/utils"

describe("Test Home component", () => {
	function Element() {
		return (
			<TestProvider>
				<Home />
			</TestProvider>
		)
	}

	it("Should render correctly", () => {
		const { container } = render(<Element />)
		expect(container).toMatchSnapshot()
	})
})
