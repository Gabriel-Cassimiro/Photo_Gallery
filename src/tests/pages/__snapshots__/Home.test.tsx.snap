// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home component Should render correctly 1`] = `
<div>
  <div
    class=" py-2 px-12 "
  >
    <div
      class="flex justify-center items-center"
    >
      <div
        class="animate-spin rounded-full h-32 w-32 border-b-2 border-gray-900"
      />
    </div>
  </div>
</div>
`;
